src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -633,7 +633,13 @@ impl ThinPool {
         }
 
         let overall_used_pct = used_pct(*used, *used + *available);
-        info!("Data tier percent used: {}", overall_used_pct);
+        debug!("Data tier percent used: {}", overall_used_pct);
```

Comment:
@waltdisgrace Please focus on giving this message more useful information. There are some models in the file that should help. I think that the other messages, which are legitimate info messages, contain a lot more information. The issue here is that stratisd can have multiple pools at a given time. This message is too bare, if it appears in the logs the person scrutinizing the logs will have to guess what pool the particular data tier belongs to.